src/test/java/core/basesyntax/services/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.services.FileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterServiceImplTest {
+    private static final String CORRECT_FILE_PATH = "src/test/resources/report.csv";
+    private static final String CORRECT_TEST_REPORT = "src/test/resources/correct_report.csv";
+    private static final String INCORRECT_FILE_PATH = "////";
+    private static final String INCORRECT_REPORT = null;
+    private static final String CORRECT_REPORT = "fruit,quantity\n"
+            + "banana, 152\n"
+            + "apple, 90";
+    private static FileWriterService fileWriterService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileWriterService = new FileWriterServiceImpl();
+    }
+
+    @Test
+    public void write_CorrectReportAndFilePath() {
+        fileWriterService.write(CORRECT_REPORT, CORRECT_FILE_PATH);
+        List<String> expected;
+        try {
+            expected = Files.readAllLines(Path.of(CORRECT_TEST_REPORT));
```

Comment:
if you write correct report to file why not use variable CORRECT_REPORT as expected ?

Reply:
Changed expected to a predefined List<Strings>